worker-workflow/src/main/resources/add-failures.js

```diff
@@ -59,7 +59,7 @@ function addFailures (document, failures, extractSourceCallback, action) {
             WORKFLOW_NAME: document.getRootDocument().getField("CAF_WORKFLOW_NAME").getStringValues().get(0),
             MESSAGE: f.getFailureMessage(),
             DATE: new Date().toISOString(),
-            CORRELATION_ID: document.getCustomData("correlationId") || undefined
+            CORRELATION_ID: document.getTask().getCorrelationId() || undefined
```

Comment:
This is a breaking change.  You can avoid this by using the existing `.getService()` method instead of the new `.getCorrelationId()` method.

See [darwin-worker-elastic/.../exposeMessageToWorker.js](https://gitlab.otxlab.net/cybersecurity-enterprise/cddri/app/darwin-worker-elastic/-/blob/v10.0.0/src/main/scripts/exposeMessageToWorker.js?ref_type=tags) for an example of using the `.getService()` method from JavaScript.